Automattic/wp-calypso
# Stats: Cleanup unused components

## Proposed Changes

This PR removes a few stats components and their corresponding unused styles that are no longer in use since the following PRs have been merged:
* #71668
* #74495
* #75996

## Testing Instructions

Verify the removed code is not in use. Calypso should build fine and there should be no bundle size changes.
